java-security/src/main/java/com/sap/cloud/security/servlet/IasTokenAuthenticator.java

```diff
@@ -34,4 +50,13 @@ protected OAuth2ServiceConfiguration getServiceConfiguration() {
 	protected OAuth2ServiceConfiguration getOtherServiceConfiguration() {
 		return null;
 	}
+
+	@Override
+	final Validator<Token> getOrCreateTokenValidator() {
```

Comment:
I don't get the gist of this implementation - can you please shortly explain?

Reply:
it seems to be leftover for the default validator integration with x509 specific validators -> not for this PR